validation/policies/io/konveyor/forklift/vmware/disk_mode.rego

```diff
@@ -0,0 +1,16 @@
+package io.konveyor.forklift.vmware
+import future.keywords.in
+
+independent_disk {
+    some i
+    input.disks[i].mode in ["independent_persistent", "independent_nonpersistent"]
+}
+
+concerns[flag] {
+    independent_disk
+    flag := {
+        "category": "Warning",
```

Comment:
is it just a warning? or critical?
can we migrate these kind of disks?

seeing the message - we can't, i think it should be critical. once a VDDK will support it, we can switch to warning.

Reply:
the category is set to Warning because it is possible to migrate such disks without VDDK (which will be significantly slower) or presumably with older versions of VDDK

Reply:
maybe it worth a mention somewhere(not only OVA workaround)? even a link to docs?

Reply:
that's already mentioned implicitly: Independent disks cannot be transferred using **recent** versions of VDDK
about link to docs, I'm not sure which docs you refer to - but the idea here is to recommend changing the disks to Dependent, that's the best resolution